src/bin/pgcopydb/ld_transform.c

```diff
@@ -43,11 +44,44 @@ typedef struct TransformStreamCtx
 	uint64_t currentMsgIndex;
 } TransformStreamCtx;
 
+static bool stream_transform_stream_internal(StreamSpecs *specs);
+
+static bool stream_transform_from_queue_internal(StreamSpecs *specs);
+
 static bool canCoalesceLogicalTransactionStatement(LogicalTransaction *txn,
 												   LogicalTransactionStatement *new);
 static bool coalesceLogicalTransactionStatement(LogicalTransaction *txn,
 												LogicalTransactionStatement *new);
 
+/*
+ * stream_transform_init_context_pgsql initializes StreamContext's
+ * transformPGSQL and opens a connection to the target. This is required to use
+ * PQescapeIdentifier API of libpq when escaping identifiers
+ */
+bool stream_transform_init_context_pgsql(StreamSpecs *specs)
```

Comment:
I wonder if the name is the best we can find. I think `stream_transform_context_init_pgsql` is better to me.